pipe/logjam/logjam_load_files.yaml

```diff
@@ -22,7 +22,7 @@ input:
   pfs:
     name: IN_PATH
     repo: logjam_list_files
-    glob: /*/*/*/logjam_prod/21115/*
+    glob: /*/*/*/logjam_prod/(21115|23646)/*
```

Comment:
FYI Several pipeline specs in here look to still be in testing mode (hard coded with specific glob pattern)